src/App.tsx

```diff
@@ -1,18 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [total] = useState(42);
+  const [perPage, setPerPage] = useState(5);
```

Comment:
Create an array of possible perPage options. And use it for select options as well